src/macros.rs

```diff
@@ -305,9 +305,9 @@ macro_rules! impl_serde_jwa {
             where
                 D: serde::Deserializer<'de>,
             {
-                let name = <&str as serde::Deserialize>::deserialize(deserializer)?;
+                let name = <alloc::borrow::Cow<'_, str> as serde::Deserialize>::deserialize(deserializer)?;
```

Comment:
Just noticed it now, using `Cow` is useless since `Cow` is always deserialized as the `Owned` variant. So, we can just use `String` directly